src/ng/directive/ngOptions.js

```diff
@@ -728,6 +728,39 @@ var ngOptionsDirective = ['$compile', '$parse', function($compile, $parse) {
         });
         removeExcessElements(currentElement);
 
+        ngModelCtrl.$validators.required = function(modelValue, viewValue) {
+
+          if (!attr.required) {
+            return true;
+          } else if (ngModelCtrl.$isEmpty(viewValue)) {
+            return false;
+          }
+
+          var isValidOption = true;
+          var viewOptions = [];
+          // Get all option and add them to viewOptions array
+          angular.forEach(options.items, function(item) {
+            viewOptions.push(options.getViewValueFromOption(item));
+          });
+
+          // In case of multiple view model is an array so validate all view values
+          // if one of them doesn't match set isValidOption to false
+          if (multiple) {
```

Comment:
I don't think we want this behavior with `multiple` the required error should only be set if no option is selected. If it's multiple and a at least option is selected, then the required error doesn't make much sense. Maybe we should actually introduce a new kind of error called `optionMismatch` or something like that.